tuesdays-with-shani-2pm-4pm/README.md

```diff
@@ -6,7 +6,9 @@ July 7th is the first Scheduled TA session with Dan
 
 Add your name to request a ticket for admittance
 
- -
+ 1. Nick Lavery
```

Comment:
Nice job with the comment about what your expectations are. I will leave the final approvals to the TAs for the requested sessions. Final approver can merge - either @Shanifos or @sashgems